packages/api/test/e2e/promise-alex-archive.spec.ts

```diff
@@ -2,10 +2,13 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
-import Api from '../../src/promise';
+import { ApiPromiseInterface } from '@polkadot/api/promise/types';
+import { Extrinsic } from '@polkadot/types';
+
+import Api from '@polkadot/api/promise';
```

Comment:
it's my personal opinion, but I prefer relative imports in tests, so that we know in a glance what we are testing

Reply:
I tried to make it a bit more consistent. We're mostly // > 50% using  the `@polkadot/...` imports in the other files, that's why I changed it in the test files, too.

Reply:
I believe the rule of thumb is:
- all imports from the same package should be relative
- all imports from other packages should be absolute

Reply:
Changed it to match that pattern.